src/main/java/careconnect/logic/Messages.java

```diff
@@ -13,6 +13,8 @@
 public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
+
+    public static final String MESSAGE_LOG_DATE_IN_FUTURE = "The log date must be in the past";
```

Comment:
```suggestion
    public static final String MESSAGE_LOG_DATE_IN_FUTURE = "The log date cannot be in the future";
```

Reply:
we allow log date to be in the present

Reply:
Wait hold on, log dates must be in the past right, because it's logs of what has already happened?

Reply:
i think its a very minor semantic issue haha. because your condition is`date.after(currentDate)` so the error is thrown only if the log date is AFTER the currentDate. This means we allow log dates which are before or on the currentDate

Reply:
i agree with MJ, very keen eye for detail 👍

Reply:
Oooh that's what you meant, yeah that makes sense hahaha